jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -43,8 +43,11 @@
 
 	private static void init() {
 		if (!initComplete) {
-			if (loadNativeLibrary()) {
-				criuSupportEnabled = isCRIUSupportEnabledImpl();
+			String isCRIUCapable = com.ibm.oti.vm.VM.getVMLangAccess().internalGetProperties().getProperty("org.eclipse.openj9.criu.isCRIUCapable"); //$NON-NLS-1$
```

Comment:
a user can set the property so its not safe to check if the JDK isnt criu enabled

Reply:
Removed it since `CRIU_SUPPORT` guards `loadNativeLibrary`.